sqswatcher/sqswatcher.py

```diff
@@ -122,87 +118,106 @@ def pollQueue(scheduler, q, t, proxy_config):
                 log.debug("SQS Message %s" % message_attrs)
 
                 try:
-                    eventType = message_attrs.get('Event')
+                    event_type = message_attrs.get('Event')
                 except:
                     try:
-                        eventType = message_attrs.get('detail-type')
+                        event_type = message_attrs.get('detail-type')
                     except KeyError:
-                        log.warn("Unable to read message. Deleting.")
+                        log.warning("Unable to read message. Deleting.")
                         message.delete()
                         break
 
-                log.info("eventType=%s" % eventType)
-                if eventType == 'autoscaling:TEST_NOTIFICATION':
+                log.info("eventType=%s" % event_type)
```

Comment:
`log.info("event_type=%s"` to match new case

Reply:
done